Source/WEB_ROOT/admin/asset/assets.html

```diff
@@ -23,11 +24,34 @@ <h1>Asset Management</h1>
 		<li><a href="vendors.html">Vendors</a></li>
 	</ul>
 
+<div>
+	<input type="text" id="searchBox" name="searchBox">
+	<button id="searchBoxBtn">Search</button>
+</div>
+
+<!-- This is the start of the main IF statement -->
+~[if#pageaction.~(gpv.action)=]
+
+<script type="text/javascript" language="JavaScript">

```

Comment:
I would likely move this to the bottom of the page, just to ensure your DOM is loaded and the javascript can easily find your objects.